package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "steam-user-tags",
+  "version": "1.0.0",
+  "description": "Display a summary view of user tags, as well as a modal popup for the full list ",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server",
+    "seed" : "node db/seed.js",
+    "build": "webpack -d --watch --mode production",
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
What testing framework are you using? I don't see it in the devDependencies.

Reply:
I'm still setting that up - that is the placeholder line for running those tests when they're ready.